gridappsd/loghandler.py

```diff
@@ -1,25 +1,21 @@
-import json
-from logging import CRITICAL, FATAL, ERROR, WARNING, WARN, INFO, DEBUG, NOTSET
+from logging import FATAL, ERROR, WARN, INFO, DEBUG, NOTSET
```

Comment:
WARNING and WARN are acceptible for the same thing.  WARN is deprecated in python.  We should not use WARN.  If we want we can leave WARN.  Why are we removing all of these?

Reply:
CRITICAL and WARN are not valid log levels in LogMessage.java or log4j.  So if you pass those to LogManager it would complain of NullPointer error. If WARN is deprecated then we have to use WARNING in gridappsd-python but when sending to the platform convert that to WARN as there is no WARNING log4j log level.

Reply:
@craig8  I pushed a change to use WARNING as level and WARN as its name. So in gridappsd-python, we can use WARNING but it sends WARN to the platform.  I tested it with 123 node model and it is working for me.

Reply:
Even though earn is deprecated we still need to handle warn.. Add the warm back in as well and this is golden